pull-requests.md

```diff
@@ -10,12 +10,19 @@ This last aspect is especially important in mono-repos as:
 - Multiple teams might be submitting changes to the repository
 - Some code changes may have non-obvious impact. For example, whether a code change directly or indirectly impacts data security. Understanding the impact of a change can be hard without the insight and input of a subject matter expert.
 
+Code review also allows for knowledge sharing in both directions between code authors and reviewers at all levels of experience. Reviewing PRs is a great way to understand a new codebase, and to stay in touch with what's happening in your team. And although a review might not be the best place for long discussions, reviews can help to surface new patterns and techniques for both code authors and reviewers.
```

Comment:
This is really good to point out as part of our culture! Nice.

Are we saying that asking questions in a review when you might want to clarify understanding it a good thing (I think so)? If so I wonder if we could outline some positive actions a person might take?

e.g.
* If you aren't clear on a piece of code, ask for clarity to solidify your understanding
* Ask if the author has time to pair with you on a runthrough of the PR

Maybe worth noting for the authors that this is never blocking as we are aware that delivery speed is important - but something they might come back to after deploying.